woocommerce_scripts/get_products_under_x.py

```
@@ -0,0 +1,93 @@
+"""
+script that will use the Woocoomerce API to
+get prices of all products and
+create a list of products that have prices less than $X.
+
+"""
+
+import sys
+import csv
+from woocommerce import API
+import os
+
+
+try:
+    woo_key = os.environ['WOO_KEY']
+    woo_secret = os.environ['WOO_SECRET']
+except KeyError:
+    raise Exception(f"The environment variables 'WOO_KEY' and 'WOO_SECRET' must be set. ")
+
+# Check if a command line argument for target price is provided
+if len(sys.argv) != 2 or len(sys.argv[1]) > 3:
+    print("Usage: python script.py <price> [where <price> is <= 6 characters long]")
+    sys.exit(1)
+
+try:
+    user_price = float(sys.argv[1])
+except ValueError:
+    print("Error: Invalid price. Please provide a whole number.")
```

same here. replace the print with 'raise Exception(....)`
You could raise a value error instead of generic exception. `raise ValueError(.....)`